lib/models.ts

```diff
@@ -23,25 +29,35 @@ export interface Config {
     width: string; // in %
 }
 
+export type StringTable = string[][]
+
 export interface InitParams {
-    inputMappings: InputMapping[][];
-    datasetSample: File;
-    callback: (cdeFileMapping: File | null) => void;
-    repositories: Repository[];
-    config: Config;
-    labName: string
-}
+    // First row should be the header.
+    // The header should have the columns in the following order:
+    // Variable Name; Abbreviation; InterlexId; Any other column.
+    // The names can vary, but the content should match with what is said above.
+    datasetMapping: StringTable;
```

Comment:
Is the datasetMapping optional? If so, could you confirm me that, for those cases, we need to use the CDE attributes as headers?

Reply:
@ddelpiano see the question of Afonso. 

I think what's meant here is: could we start doing he mapping without a dataset mapping file?
In other words: will a default (initial) dataset mapping file be created on the first step if there is no dataset mapping file specified

@afonsobspinto imho it shouldn't matter, if there is no dataset mapping file specified we can create an empty one. or is there a technical reason not to do it